mail_debrand/README.rst

```diff
@@ -20,7 +20,7 @@ Mail Debrand
     :target: https://runbot.odoo-community.org/runbot/205/13.0
     :alt: Try me on Runbot
 
-|badge1| |badge2| |badge3| |badge4| 
+|badge1| |badge2| |badge3| |badge4|
```

Comment:
You should touch these files manually...

Reply:
@pedrobaeza agreed. But for all versions? Perhaps in older versions the bot does not auto generate the readme. Not sure.

Reply:
Yes, everywhere is updated. If edited manually, you must removed the automatic trim of trailing whitespaces in the IDE.

Reply:
@pedrobaeza A new .rst file will be autogenerated independently if I touch the file or not. So, why bother if I touch it or not?

Reply:
Because it generates a lot of diff noise.

Reply:
The diff noise will exist independently of what I do. We can't avoid the noise.

Reply:
@pedrobaeza the diff noise was removed. could we merge this?